alica_engine/CMakeLists.txt

```diff
@@ -10,16 +10,16 @@ find_package(yaml-cpp REQUIRED)
 ##ROS/Catkin
 find_package(catkin QUIET)
 if (catkin_FOUND)
-  find_package(catkin REQUIRED COMPONENTS id_manager fsystem event_handling system_util alica_common_config alica_solver_interface)
+  find_package(catkin REQUIRED COMPONENTS id_manager fsystem event_handling system_util alica_common_config alica_solver_interface amr_interfaces)
```

Comment:
alica should not depend on amr_interfaces. I assume this is needed for the trace context. In that case, we simply use std::string directly.